README.md

```diff
@@ -3,123 +3,27 @@
+Want to contribute to HTM? Consider the [contibution workflow](https://h-transport-materials.readthedocs.io/en/latest/dev/contributing.html) in the docs.
 
+Alternatively if you have a material property you would like to add
 
-plt.yscale("log")
-plt.ylabel("Diffusivity (m$^2$/s)")
-plt.legend()
-plt.show()
-```
-![Figure_1](https://user-images.githubusercontent.com/40028739/213814746-9dadb8dc-599e-4004-8135-e496b19fd8bc.png)
+Spot a bug! or have any suggestions, feel free to raise an
+[issue](https://github.com/RemDelaporteMathurin/h-transport-materials/issues/new).
 
->
-### Add custom properties
 
-```python
-import h_transport_materials as htm
+# Citing h-transport-materials
 
-import numpy as np
-
-# Create a custom property
-my_custom_property = htm.ArrheniusProperty(pre_exp=1e-5, act_energy=0.2)
-
-# From (T, y) data
-my_fitted_property = htm.ArrheniusProperty(
-    data_T=[300, 400, 500, 600],
-    data_y=[1e-8, 1e-7, 1e-6, 1e-5],
-)
-
-print("Pre-exponential factor: {:.2e}".format(my_fitted_property.pre_exp))
-print("Activation energy: {:.2f} eV".format(my_fitted_property.act_energy))
-
-# Pre-exponential factor: 4.40e-03
-# Activation energy: 0.35 eV
-```
-
-### Filters
-
-```python
-import h_transport_materials as htm
-
-# tungsten solubilities
-htm.solubilities.filter(material="tungsten")
-
-# copper and cucrzr solubilities
-htm.solubilities.filter(material=["copper", "cucrzr"])
-
-# all_authors_except_ryabchikov
-htm.diffusivities.filter(material="tungsten").filter(author="ryabchikov", exclude=True)
-
-# only Tritium
-htm.diffusivities.filter(isotope="t")
-```
-
-### Compute mean values
-
-```python
-import h_transport_materials as htm
-import matplotlib.pyplot as plt
-
-tungsten_diffusivities = htm.diffusivities.filter(material="tungsten").filter(
-    author=["moore", "zakharov"], exclude=True
-)
-
-# compute mean diffusivity
-mean_diffusivity = tungsten_diffusivities.mean()
-
-# plot
-htm.plotting.plot(tungsten_diffusivities, alpha=0.5)
-
-htm.plotting.plot(mean_diffusivity, color="black", linewidth=3)
-
-x_annotation = 0.0034
-plt.annotate("mean value", (x_annotation, mean_diffusivity.value(T=1 / x_annotation)))
-
-plt.ylabel("Diffusivity (m$^2$ s$^{-1}$)")
-plt.yscale("log")
-plt.show()
-```
-![Figure_1](https://user-images.githubusercontent.com/40028739/169285178-7cccc183-8ae1-4afe-8e4e-af2d54ac8741.svg)
-
-### Export to BibTeX
-
-The refernces of properties and properties groups can be exported to a bibfile.
-
-```python
-import h_transport_materials as htm
-
-tungsten_diffusivities = htm.diffusivities.filter(material="tungsten")
-
-tungsten_diffusivities.export_bib("my_bibfile.bib")
-
-```
-
-
-# Contributions
-
-The current database is far from complete.
-[Contributions](https://github.com/RemDelaporteMathurin/h-transport-materials/issues/new) are most welcome to extend it by adding new properties and also new materials!
+If you have used h-transport material, please consider aknowlegdement of your usage by:
```

Comment:
Can we have a link to the Zenodo DOI? or maybe the BibTex version?

Reply:
What do you think of the option added. maybe the Bibtex file should be in a txt file or we could add a CITATION.cff?